src/context/metrikaContext/metrikaContext.tsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
-import {Metrika, Pixel} from '../../models';
+import {GaInline, Metrika, Pixel} from '../../models';
 
 export interface MetrikaContextProps {
     metrika?: Metrika;
     pixel?: Pixel;
+    gaInline?: GaInline;
```

Comment:
here and anywhere else gaInline -> ga or googleAnalytics

Reply:
ok